README.md

```diff
@@ -202,14 +202,21 @@ The following table lists the configurable parameters of the Concourse chart and
 | `web.replicas` | Number of Concourse Web replicas | `1` |
 | `web.resources.requests.cpu` | Minimum amount of cpu resources requested | `100m` |
 | `web.resources.requests.memory` | Minimum amount of memory resources requested | `128Mi` |
-| `web.service.annotations` | Concourse Web Service annotations | `nil` |
-| `web.service.atcNodePort` | Sets the nodePort for atc when using `NodePort` | `nil` |
-| `web.service.atcTlsNodePort` | Sets the nodePort for atc tls when using `NodePort` | `nil` |
-| `web.service.labels` | Additional concourse web service labels | `nil` |
-| `web.service.loadBalancerIP` | The IP to use when web.service.type is LoadBalancer | `nil` |
-| `web.service.loadBalancerSourceRanges` | Concourse Web Service Load Balancer Source IP ranges | `nil` |
-| `web.service.tsaNodePort` | Sets the nodePort for tsa when using `NodePort` | `nil` |
-| `web.service.type` | Concourse Web service type | `ClusterIP` |
+| `web.service.atc.annotations` | Concourse Web ATC Service annotations | `nil` |
+| `web.service.atc.NodePort` | Sets the nodePort for atc when using `NodePort` | `nil` |
+| `web.service.atc.tlsNodePort` | Sets the nodePort for atc tls when using `NodePort` | `nil` |
+| `web.service.atc.labels` | Additional concourse web atc service labels | `nil` |
+| `web.service.atc.loadBalancerIP` | The IP to use when web.service.atc.type is LoadBalancer | `nil` |
+| `web.service.atc.loadBalancerSourceRanges` | Concourse Web ATC Service Load Balancer Source IP ranges | `nil` |
+| `web.service.atc.type` | Concourse Web ATC service type | `ClusterIP` |
+| `web.service.tsa.annotations` | Concourse Web TSA Service annotations | `nil` |
```

Comment:
```suggestion
| `web.service.workerGateway.annotations` | Concourse Web TSA Service annotations | `nil` |
```

Same as previous comment, `tsa` to `workerGateway`

Reply:
Hi @taylorsilva, i renamed the services as suggested. I tied to run the tests locally on my machine, but is not easy. I could not get the clean results for `master`. My machine is a Mac with 16GB RAM.

Reply:
Hi @taylorsilva,
I created concourse/concourse#5665 to fix the topgun tests. I was able to run the tests locally.

Reply:
yay! Glad you got it working! I'm curious, were you using a local k8s cluster as well like [kind](https://kind.sigs.k8s.io/) or an external one?

Reply:
I used a local kind cluster. I had to tweak the docker ressources and needed to close almost all windows. Used the following kind.config
```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
- role: worker
- role: worker
```